scripts/msct_register.py

```diff
@@ -45,13 +47,27 @@ def register_slicewise(fname_src,
                         remove_temp_files=0,
                         verbose=0):
 
+    im_and_seg = (paramreg.algo == 'centermassrot') and (paramreg.rot_method != 'PCA')  # bool for simplicity
```

Comment:
instead, how about:
~~~
 im_and_seg = (paramreg.algo == 'centermassrot') and (paramreg.rot_method == 'HOG') 
~~~
to make it explicit in the code that `im_and_seg` should be used with HOG method.

Reply:
I was gong with the !=PCA approach instead someone, one day (maybe me) wants to try another method that needs both im and seg

Reply:
then it would look like:
~~~
im_and_seg = (paramreg.algo == 'centermassrot') and ( (paramreg.rot_method == 'HOG') or (paramreg.rot_method == 'OTHER_METHOD') )
~~~

Reply:
otherwise, the OTHER_METHOD will automatically use im_and_seg, which is not necessarily true

Reply:
I agree, let's go with that :+1: